EventListener/UserContextSubscriber.php

```diff
@@ -61,21 +62,28 @@ class UserContextSubscriber implements EventSubscriberInterface
      */
     private $ttl;
 
+    /**
+     * @var \Symfony\Component\HttpFoundation\RequestMatcherInterface
```

Comment:
please only use RequestMatcherInterface here, `use` statements work for annotations too